lightning/src/chain/channelmonitor.rs

```diff
@@ -1670,25 +1670,25 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitor<Signer> {
 		);
 	}
 
-	/// Returns the descriptor for a relevant output (i.e., one that we can spend) within the
-	/// transaction if one exists and the transaction has at least [`ANTI_REORG_DELAY`]
+	/// Returns the descriptors for relevant outputs (i.e., those that we can spend) within the
+	/// transaction if they exist and the transaction has at least [`ANTI_REORG_DELAY`]
 	/// confirmations.
 	///
 	/// Descriptors returned by this method are primarily exposed via [`Event::SpendableOutputs`]
 	/// once they are no longer under reorg risk. This method can serve as a way to retrieve these
 	/// descriptors at a later time, either for historical purposes, or to replay any
 	/// missed/unhandled descriptors.
 	///
-	/// `tx` is a transaction we'll scan the outputs of. If an output which can be spent by us is
-	/// found, a descriptor is returned. `confirmation_height` must be the height of the block in
-	/// which `tx` was included in.
-	pub fn get_spendable_output(&self, tx: &Transaction, confirmation_height: u32) -> Option<SpendableOutputDescriptor> {
+	/// `tx` is a transaction we'll scan the outputs of. If any outputs which can be spent by us are
+	/// found, their descriptors are returned. `confirmation_height` must be the height of the block
+	/// in which `tx` was included in.
```

Comment:
May be worth noting that this will ~always return 0 or 1 spendable output unless something weird's happening

Reply:
Hmm. I don't think it's really useful to the user to mention that. They should always handle the multiple spendable outputs case. But I guess it can't hurt to mention with short explanation.